tests/acceptance/features/bootstrap/CapabilitiesContext.php

```diff
@@ -39,81 +39,16 @@ class CapabilitiesContext implements Context {
 	 */
 	private $featureContext;
 
-	/**
-	 * @Then the capabilities should contain
-	 *
-	 * @param TableNode|null $formData
-	 *
-	 * @return void
-	 * @throws Exception
-	 */
-	public function checkCapabilitiesResponse(TableNode $formData):void {
-		$capabilitiesXML = $this->featureContext->appConfigurationContext->getCapabilitiesXml(__METHOD__);
-		$assertedSomething = false;
-
-		$this->featureContext->verifyTableNodeColumns($formData, ['value', 'path_to_element', 'capability']);
-
-		foreach ($formData->getHash() as $row) {
-			$row['value'] = $this->featureContext->substituteInLineCodes($row['value']);
-			Assert::assertEquals(
-				$row['value'] === "EMPTY" ? '' : $row['value'],
-				$this->featureContext->appConfigurationContext->getParameterValueFromXml(
-					$capabilitiesXML,
-					$row['capability'],
-					$row['path_to_element']
-				),
-				"Failed field {$row['capability']} {$row['path_to_element']}"
-			);
-			$assertedSomething = true;
-		}
-
-		Assert::assertTrue(
-			$assertedSomething,
-			'there was nothing in the table of expected capabilities'
-		);
-	}
-
-	/**
-	 * @Then the version data in the response should contain
-	 *
-	 * @param TableNode|null $formData
-	 *
-	 * @return void
-	 * @throws Exception
-	 */
-	public function checkVersionResponse(TableNode $formData):void {
-		$versionXML = $this->featureContext->appConfigurationContext->getVersionXml(__METHOD__);
-		$assertedSomething = false;
-
-		$this->featureContext->verifyTableNodeColumns($formData, ['name', 'value']);
-
-		foreach ($formData->getHash() as $row) {
-			$row['value'] = $this->featureContext->substituteInLineCodes($row['value']);
-			$actualValue = $versionXML->{$row['name']};
-
-			Assert::assertEquals(
-				$row['value'] === "EMPTY" ? '' : $row['value'],
-				$actualValue,
-				"Failed field {$row['name']}"
-			);
-			$assertedSomething = true;
-		}
-
-		Assert::assertTrue(
-			$assertedSomething,
-			'there was nothing in the table of expected version data'
-		);
-	}
-
 	/**
 	 * @Then the major-minor-micro version data in the response should match the version string
 	 *
 	 * @return void
 	 * @throws Exception
 	 */
 	public function checkVersionMajorMinorMicroResponse():void {
```

Comment:
isn't that checked by 
```
                    "versionstring": {
                      "type": "string",
                      "enum": ["%versionstring%"]
                    }
```
I don't understand why we even have that step, I think the expectation AND the result comes from the same response

Reply:
we are actually asserting information from this `version` with  `versionstring` from the same response
```
"version": {
                "major": 10,
                "minor": 11,
                "micro": 0,
                "string": "10.11.0",
                "edition": "Community",
                "product": "Infinite Scale",
                "productversion": "2.0.0+22f3577b7"
            } ```